align.go

```diff
@@ -0,0 +1,115 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plot
+
+import (
+	"math"
+
+	"github.com/gonum/plot/vg"
+	"github.com/gonum/plot/vg/draw"
+)
+
+// Align returns a two-dimensional row-major array of Canvases which will
+// produce tiled plots with DataCanvases that are evenly sized and spaced.
+// The arguments to the function are a two-dimensional row-major array
+// of plots, a tile configuration, and the canvas to which the tiled
+// plots are to be drawn.
+func Align(plots [][]*Plot, t draw.Tiles, dc draw.Canvas) [][]draw.Canvas {
```

Comment:
I know "facet" is the word ggplot uses to describe these, so it could be useful to maintain continuity with that, but on the other hand "facet" is really referring to the data that's being plotted, whereas what this function is actually doing is "align"ing. My slight preference is "align" but I would be fine with either one.